### ISSUE TITLE
Rename default “docs” root to “pages”?

### ISSUE THREAD

**Issue by mbostock** [observablehq/framework#752]
Since documentation is not our primary use case, “docs” isn’t a great default. But maybe it’s too disruptive to change now.

**Comment by mhalle** [comment]
As a new user scanning the directory, I was immediately confused by "docs" and interpreted the contents as documentation. +1 for "pages" or some other alternative.

**Comment by mbostock** [comment]
We could start by changing the default templates to use `pages` instead of `docs`.

**Comment by mythmon** [comment]
I made a small attempt at changing the default in the templates. That was simple, of course, but I started having trouble with rewriting some docs. We refer to that directory, either by name or as a component in file paths, in several places. I didn't have the right language to rewrite those in a way that didn't seem confusing. I pushed up my in-progress branch if anyone else wants to attempt docs: https://github.com/observablehq/framework/tree/mythmon/240325/default-root-pages
